### ISSUE TITLE
Please add arm support

### ISSUE THREAD

**Issue by up2smoke** [NullArray/AutoSploit#1221]
# Running information

 
 - What branch did you download?
 - Clone, or docker run? Both
 - What OS are you running? Raspbian Lite / Raspberry Pi 3B+

# Exploit module information

 
 - What exploit was deployed?
 - Was a session generated for the target?
 - What version of metasploit are you running?

# Program information

 
 - Python version number?
 - AutoSploit version number?
 - Any console output that is relevant to the issue:
 - Traceback (error) if any:

pi@raspberrypi:~/AutoSploit/Docker $ sudo docker build -t autosploit ./
Sending build context to Docker daemon  8.192kB
Step 1/6 : FROM phocean/msf
 ---> 88a42fa935f3
Step 2/6 : COPY "entrypoint.sh" .
 ---> Using cache
 ---> 2e34200ce2d2
Step 3/6 : RUN apt-get update &&        apt-get install -y      git                                      python-dev              python-pip                                       apache2
 ---> Running in 2a5a76004227
standard_init_linux.go:211: exec user process caused "exec format error"
The command '/bin/sh -c apt-get update &&       apt-get install -y                                       git                     python-dev                                       python-pip      apache2' returned a non-zero code: 1
pi@raspberrypi:~/AutoSploit/Docker $

**Comment by Ekultek** [comment]
are you using raspian, or did you flash it with another kernel?

**Comment by up2smoke** [comment]
raspbian Lite 
metasploit-frameworks works

**Comment by Ekultek** [comment]
@up2smoke i'm gonna have to get a pi for testing purposes so this might take a bit to fix, unless you know of software that i can mimic one with?

**Comment by Ekultek** [comment]
I figured something out using qemu and rasp lite, I'll start working on this asap because it sounds fucking awesome and im excited to do it lol

**Comment by up2smoke** [comment]
nice to hear that :) unfortunately I don't know any software to emulate a Raspberry Pi.  but if that works with qemu, I would be happy to test it for you.  I have an rPi 3B + & rPi ZeroW

**Comment by NullArray** [comment]
To the best of my recollection you can actually emulate Rasbian with KVM-QEMU. Virt-Manager will even allow you to set the hardware specs via GUI if you are so inclined.

**Comment by up2smoke** [comment]
Any Updates? U See the previous comments are ddeleted?

**Comment by NullArray** [comment]
> Any Updates? U See the previous comments are deleted?

Yeah they're deleted. Unfortunately Ekultek nuked his Github presence from orbit. What that means for AutoSploit specifically is that we lost a very active contributor. In practical terms this will cause some delays with regards to updates and such since i am maintaining the repo on my own now.

Do you recall what the other comments were about? Aside from emulating Rasbian? I can't undelete them or i would have. Since it also breaks continuity in discussions, as evidenced here.

Sorry for the inconvenience.

**Comment by NullArray** [comment]
Tell you what, i was messing with QEMU anyway, i can try running Rasbian + AS and report back with any interesting findings, but i can't promise you i'll have time to fix everything today.

How about it?

**Comment by up2smoke** [comment]
Nothing importend:
Ekultek 15.12:
`are you using raspian, or did you flash it with another kernel?`
Ekultek 16.12:
`@up2smoke i'm gonna have to get a pi for testing purposes so this might take a bit to fix, unless you know of software that i can mimic one with?`
Ekultek 16.12
`I figured something out using qemu and rasp lite, I'll start working on this asap because it sounds fucking awesome and im excited to do it lol'


I'm sorry to hear that.  AutoSploit is a great project!  Unfortunately I am not a developer and I have no idea about coding

**Comment by Ekultek** [comment]
Working on it, been busy lately. Sorry for the disappearing act

**Comment by NullArray** [comment]
> Working on it, been busy lately. Sorry for the disappearing act

All good.

**Comment by up2smoke** [comment]
nice to see you back here.  happy New Year

**Comment by up2smoke** [comment]
Any update?

**Comment by up2smoke** [comment]
I found a solution how to run it on a raspberry pi 3b+ & 4b with raspbianLite OS.
The reason is, to just install "libffi-dev" before. I close this issue now

**Comment by up2smoke** [comment]
Any update maybe?
